server/db/models/product.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+var Sequelize = require('sequelize');
+
+module.exports = function (db) { 
+	db.define('product', {
+		name: {
+			type: Sequelize.STRING,
+			allowNull: false,
+            unique: true
+		},
+		imageUrl: {
```

Comment:
imageUrl might not be as descriptive as imageUrl 's', since it will be an array.